server/service/common.go

```diff
@@ -6,4 +6,5 @@ import "database/sql"
 type SQLDB interface {
```

Comment:
as I understand we need this interface only for testing purposes.
any particular reason to mock it on this level instead of creating mocked `sql.DB`?

Reply:
discussed offline:
- we call `.Close()` on db so just `&sql.DB{}` wouldn't work
- bringing any mocking dependency is out of scope for now
- though @dpordomingo is unhappy with this interface too :)